framework/bug-mining/vcs-log-xref.pl

```diff
@@ -32,12 +32,16 @@ =head1 NAME
 
 =head1 SYNOPSIS
 
-vcs-log-xref.pl -e bug_matching_perl_regexp -l commit_log -r repository_dir -i issues_file -f output_file [-v vcs_type] [-n project_name]
+vcs-log-xref.pl -p project_id -e bug_matching_perl_regexp -l commit_log -r repository_dir -i issues_file -f output_file [-v vcs_type] [-n project_name]
```

Comment:
I suggest to keep the notion of existing vs. new projects out of the lower-level helper scripts and rather encode the different behavior in the higher-level scripts. Are there any advantages of the current solution that I am missing?

Reply:
Good point. Yes, we should keep that in the higher-level scripts.